### PR TITLE
Include Swarm bzz_ methods

### PR DESCRIPTION
**Pull request by axic**
This is work in progress and depends on the progress of the Swarm integration in geth.  Comments welcome.

cc @zelig

### CODE REVIEW

File: lib/web3/methods/swarm.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+ * @date 2016
+ *
+ * Reference: https://github.com/ethereum/go-ethereum/blob/swarm/internal/web3ext/web3ext.go#L33
+ */
+
+"use strict";
+
+var Method = require('../method');
+var Property = require('../property');
+var formatters = require('../formatters');
```
Comment (luclu) [inline lib/web3/methods/swarm.js]: travis (jshint) complains about this required but not being used

Comment (axic) [inline lib/web3/methods/swarm.js]: fixed 👍

**Comment by rfikki** [comment]
+1

**Comment by axic** [comment]
Hooray! (And I've just checked against latest geth that the content of this PR still holds, and it does 😃 )

**Comment by MrTibbles** [comment]
👍

**Comment by iurimatias** [comment]
@axic @zelig how do we enable this namespace in geth?

**Pull request merged**